website/addons/github/model.py

```diff
@@ -76,10 +76,7 @@ def revoke_remote_oauth_access(self, external_account):
     # Assumes oldest connected account is primary.
     @property
     def public_id(self):
-        gh_accounts = [
-            a for a in self.owner.external_accounts.all()
-            if a.provider == self.oauth_provider.short_name
-        ]
+        gh_accounts = self.owner.external_accounts.filter(provider=self.oauth_provider.short_name)
         if gh_accounts:
             return gh_accounts[0].display_name
```

Comment:
@cwisecarver But this method returns the display_name of the first result in the queryset a few lines down.


The result set should be very small (# of Github accounts attached to a user), so I don't think the sort will be an issue.